<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySQLData</class>
 <widget class="QWidget" name="MySQLData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySQLData</string>
  </property>
  <widget class="QPushButton" name="BT_testsql">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>数据库显示</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>531</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>548</x>
     <y>250</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>连接名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>548</x>
     <y>290</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>数据库名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>548</x>
     <y>330</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>表名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_connectname">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MySQL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sqlname">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>290</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>voltageclass</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tablename">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>setvoltageclass</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
